MAINTAINERS

```diff
@@ -32,6 +32,11 @@ basis (R:).
 
 ----------
 
+AOSP build
+R:	Tadd Kao <tadd.kao@mediatek.com> [@taddk]
+S:	Maintained
+F:	mk/aosp_optee.mk
+
```

Comment:
That's fine but I would prefer a separate commit for this (you can add my 
`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`)

Reply:
OK.